doc/whats_new/v0.24.rst

```diff
@@ -230,12 +234,15 @@ Changelog
   :meth:`tree.DecisionTreeRegressor.fit`, and has not effect.
   :pr:`17614` by :user:`Juan Carlos Alfaro Jiménez <alfaro96>`.
 
+<<<<<<< HEAD
+=======
```

Comment:
Those two lines should be removed: they are the relic of the automatic git diff with upstream